app/controllers/monologue/controller_helpers/auth.rb

```diff
@@ -13,9 +13,10 @@ module Auth
       private
       def authenticate_user!
          if monologue_current_user.nil?
-           redirect_to monologue.admin_login_url, alert: I18n.t("monologue.admin.login.need_auth")
+           redirect_to monologue.admin_login_url, alert: I18n.t('monologue.admin.login.need_auth')
```

Comment:
Line is too long. [98/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.